cmd/macaroon/brave-skus/free_dev.yaml

```diff
@@ -0,0 +1,13 @@
+tokens:
+  - id: "brave-free sku token v1"
+    version: 1
+    location: "freetrial.bsg.brave.software"
+    first_party_caveats:
+      - sku: "brave-free"
+      - price: 0
```

Comment:
NOTE: the resulting token for any free SKUs should be treated as a secret as the order creation endpoint is public and thus can be used to create unlimited orders of said free item if the SKU token is known